### PR TITLE
VRF Lite Support - update

### PR DESCRIPTION
**Pull request by HenocKA**
Bug correction in arp-table discovery and bgp-peers, 
adding vrf-cisco-lite in device (poller)

### CODE REVIEW

**Comment by laf** [comment]
Due to the issues with original code and the fact that none of the @librenms/reviewers really has the kit to test all possible scenarios then I'd suggest that this is broken into two. BGP and ARP updates, that way we can merge them in separately giving people time to feed back.

**Comment by Tristan-Labbe** [comment]
I'm not sure to understand.
You want 2 pool request ? 
One for BGP module and an other for ARP ?

PS: I'm working with HenocKa.

**Comment by laf** [comment]
Yes please, it will be easier to merge / fix issues if we separate these out.

**Comment by Tristan-Labbe** [comment]
No problem,

you can find the pull request with ARP here : https://github.com/librenms/librenms/pull/3359

and the pull request with BGP here : https://github.com/librenms/librenms/pull/3360

**Comment by HenocKA** [comment]
So i revert arp and bgp updates from this PR.
This PR adds some changes on poller and changelog

**Comment by laf** [comment]
This PR has over 150 commits and 250 files altered, can you please squash it down to just your changes please.

File: poller.php [hunk 1]
```diff
@@ -129,11 +129,12 @@
 echo "Starting polling run:\n\n";
 $polled_devices = 0;
 if (!isset($query)) {
-    $query = "SELECT `device_id` FROM `devices` WHERE `disabled` = 0 $where ORDER BY `device_id` ASC";
+    $query = "SELECT `device_id` FROM `devices` WHERE `disabled` = 0 $where ORDER BY `device_id` ASC;";
```
Comment (laf) [inline poller.php]: Can you remove the added semi-colon please. It's not needed.

**Pull request merged**